src/extensions/npmjs-latest-version.checker.ts

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
Which file should we keep and which must be deleted?

There is `npmjs-latest-version-checker.ts` as well as `npmjs-latest-version.checker.ts`.